src/devtools/useAtomsSnapshot.ts

```diff
@@ -18,7 +21,12 @@ const createAtomsSnapshot = (
     const atomState = store[DEV_GET_ATOM_STATE]?.(atom) ?? ({} as AtomState)
     return [atom, 'v' in atomState ? atomState.v : undefined]
   })
-  return new Map(tuples)
+  const dependants = atoms.map<[Atom<unknown>, Set<Atom<unknown>>]>((atom) => {
+    const mounted = store[DEV_GET_MOUNTED]?.(atom) ?? ({} as Mounted)
+    return [atom, mounted.t]
```

Comment:
I guess you can simply do this?
```suggestion
    const mounted = store[DEV_GET_MOUNTED]?.(atom)
    return [atom, mounted?.t]
```